### ISSUE TITLE
LTL Parser cannot deal with predicate DEFINITIONS

### ISSUE THREAD

**Issue by leuschel** [hhu-stups/prob-issues#62]
The LTL parser throws parse errors when predicate DEFINITIONS are used inside.

For example, given
  GOAL == crit:ran(pc);
This formula gives a parser error
  "GF {GOAL}"

```
! An error occurred (source: ltl) !
! Syntax error:  [2,5] expecting: not belonging, non inclusion, strict non inclusion, not equal, element of, less, inclusion, strict inclusion, less equal, equal, greater, greater equal

! An error occurred (source: ltl) !
! LTL Parser failed:  GF {GOAL}
```
However, normal definitions work:

```
  RPC == ran(pc);
  ASSERT_LTL101 == "GF {crit:RPC}" // ok
```

**Comment by leuschel** [comment]
For ProB Tcl/Tk this issue has been solved in this commit of prob_parsers: f06736d1daf934dfcaec0945a9a13cb4b19c54c7

**Comment by favu100** [comment]
This issue is solved now
